rustls/src/client/client_conn.rs

```diff
@@ -249,15 +250,15 @@ impl fmt::Debug for ClientConfig {
 impl ClientConfig {
     #[cfg(feature = "ring")]
     /// Create a builder to build up the client configuration with the default
-    /// [`CryptoProvider`].
+    /// [`CryptoProvider`]: [`crate::crypto::ring::RING`].
```

Comment:
Squash these into the originating commit(s)?

Reply:
The originating commit for these is already on main (048ff1074)